dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/DSpaceRestRepository.java

```diff
@@ -406,5 +407,20 @@ protected void patch(Context context, HttpServletRequest request, String apiCate
         throws SQLException, FileNotFoundException, IOException, AuthorizeException {
         throw new RepositoryMethodNotImplementedException("No implementation found; Method not allowed!", "");
     }
+    public T put(HttpServletRequest request, String apiCategory, String model, ID uuid, JsonNode jsonNode) {
+        Context context = obtainContext();
+        try {
+            thisRepository.put(context, request, apiCategory, model, uuid, jsonNode);
+            context.commit();
+        } catch (SQLException | AuthorizeException e) {
+            throw new RuntimeException(e.getMessage(), e);
```

Comment:
Again, improve error message

Reply:
Should be something like...
`throw new RuntimeException("Unable to update DSpace object " + model + " with id=" + uuid, e);`